featuretools/entityset/entityset.py

```diff
@@ -1142,40 +1141,6 @@ def _import_from_dataframe(self,
                 Defaults to False.
         """
         variable_types = variable_types or {}
-
-        # DFS TODO: confirm we want this if else block
-        if index is None:
-            assert not make_index, "Must specify an index name if make_index is True"
-            logger.warning(("Using first column as index. ",
-                            "To change this, specify the index parameter"))
-            index = dataframe.columns[0]
-        else:
-            if index not in variable_types:
-                variable_types[index] = vtypes.Index
-
-        created_index = None
-
-        if make_index and index in dataframe.columns:
-            raise RuntimeError("Cannot make index: index variable already present")
-
-        if make_index or index not in dataframe.columns:
-            if not make_index:
-                logger.warning("index %s not found in dataframe, creating new integer column",
-                               index)
-            dataframe.insert(0, index, range(0, len(dataframe)))
-            created_index = index
-
-        if time_index is not None and time_index not in dataframe.columns:
-            raise LookupError('Time index not found in dataframe')
-
-        for c in dataframe.columns:
-            if not is_string(c):
-                raise ValueError("All column names must be strings (Column {} is not a string)".format(c))
-
-            if dataframe[c].dtype.name.find('category') > -1:
```

Comment:
in `infer_variable_types` this logic is just 

```
elif df[variable].dtype.name == "category":
      inferred_type = vtypes.Categorical
```

did this handle a case that the other didn't or were we just being extra cautious to just look for "category" anywhere in the name?

Reply:
I'm not sure.  It looks like from [this documentation](https://pandas.pydata.org/pandas-docs/stable/categorical.html#equality-semantics) that if we check for `df[variable].dtype == 'category'` instead we'll catch all CategoricalDtype.

Reply:
cool. let's refactor to that